packages/sanes-chrome-extension/src/logic/persona/persona.ts

```diff
@@ -21,6 +22,7 @@ import {
   pathBuilderForCodec,
 } from '../config';
 import { AccountManager, AccountInfo, AccountManagerChainConfig } from './accountManager';
+import { isNewSender } from './personaWhitelist';
 
 /** Like UseOnlyJsonRpcSigningServer but without functionality to create or shutdown */
 export interface UseOnlyJsonRpcSigningServer {
```

Comment:
After a better understanding and keeping in mind next steps, I think we should move the `startSigningServer` in a different class. Persona.ts is almost 200 LOC and is doing too much (complex things).

@webmaster128 @albertandrejev what do you guys think?

Reply:
`startSigningServer` is in Persona.ts because it needs to access the same MultiChainSigner instance as the Persona and I want to avoid exposing this instance. Everything in Persona fits together logically, no matter how many lines it has.

Reply:
Does not entirely true, you can wrap functionality by folder without loosing encapsulation.

If the person's continues growing when a developer arrived to this class won't understand a thing.

If the rest of the team agrees to leave it a it is it is fine with me. @abefernan @albertandrejev  @ethanfrey , what do you think?

Reply:
The callback implementation should be created somewhere else, so the `startSigningServer` can remain relativly simple. See https://github.com/iov-one/ponferrada/pull/151